tests/testsuite/build.rs

```diff
@@ -3953,8 +3953,10 @@ fn run_proper_binary_main_rs_as_foo() {
     p.cargo("run --bin foo").run();
 }
 
+// NOTE: we don't have `/usr/bin/env` on Windows.
```

Comment:
Any reason to displace this from the relevant code?
When on the same line (here and in other places) it's clearly talking about that line.  When not, it can be ambiguous if it's about the previous code or next code.

Reply:
My bad. I thought that comment applied to several methods, but on closer inspection, it doesn't.